contracts/compliance/modular/ModularCompliance.sol

```diff
@@ -111,12 +127,12 @@ contract ModularCompliance is IModularCompliance, OwnableUpgradeable, MCStorage
      *  @dev See {IModularCompliance-addModule}.
      */
     function addModule(address _module) external override onlyOwner {
-        require(_module != address(0), "invalid argument - zero address");
-        require(!_moduleBound[_module], "module already bound");
-        require(_modules.length <= 24, "cannot add more than 25 modules");
+        require(_module != address(0), InvalidArgumentErrors.ZeroAddress());
+        require(!_moduleBound[_module], ModuleAlreadyBound());
+        require(_modules.length <= 24, CannotAddMoreThan25Modules());
         IModule module = IModule(_module);
         if (!module.isPlugAndPlay()) {
-            require(module.canComplianceBind(address(this)), "compliance is not suitable for binding to the module");
+            require(module.canComplianceBind(address(this)), ComplianceNotSuitableForBindingToModule());
```

Comment:
i would add the module address as a parameter of this error, this way you know which module cannot be bound even if you try to bind a batch of modules (which is what TREX Factory does at deployment for example)